target/linux/ipq807x/files/drivers/hwmon/g761.c

```diff
@@ -0,0 +1,1098 @@
+/*
```

Comment:
This should be submitted upstream first, otherwise, it will end up being downstream forever for OpenWrt to maintain.

Reply:
Its design also looks somewhat old, so it needs to be updated to use probe_new, get rid of OF being optional, replace the license text with SPDX identifier etc.

Reply:
66147c8a3b RAX120: Remove unsupported g761 driver